src/predicate.ts

```diff
@@ -58,6 +58,104 @@ export function isFieldEqualPredicate(predicate: any): predicate is FieldEqualPr
   return predicate && !!predicate.field && predicate.equal !== undefined;
 }
 
+export interface FieldLTPredicate {
```

Comment:
Can you make all of these interfaces extend `FieldPredicateBase` so `timeUnit` and `field` don't have to be declared repeatedly?